src/fastapi_poe/base.py

```diff
@@ -429,6 +436,8 @@ async def _make_file_attachment_request(
                         "is_inline": is_inline,
                         "download_url": download_url,
                     }
+                    if download_filename:
```

Comment:
do we need to modify the raise exception above to allow for filename as param along side download URL?

Reply:
No. Since this is adding a new optional param specific for download_url only.